server/licenses/jzlib-LICENSE.txt

```diff
@@ -0,0 +1,29 @@
+JZlib 0.0.* were released under the GNU LGPL license.  Later, we have switched
+over to a BSD-style license.
+
+------------------------------------------------------------------------------
+Copyright (c) 2000-2011 ymnk, JCraft,Inc. All rights reserved.
+
+Redistribution and use in source and binary forms, with or without
+modification, are permitted provided that the following conditions are met:
+
+  Redistributions of source code must retain the above copyright notice, this
+  list of conditions and the following disclaimer.
+
+  Redistributions in binary form must reproduce the above copyright notice,
+  this list of conditions and the following disclaimer in the documentation
+  and/or other materials provided with the distribution.
+
+  The name of the authors may not be used to endorse or promote products
+  derived from this software without specific prior written permission.
+
```

Comment:
I am assuming this license is approved for usage

Reply:
Yes, it is approved. Ref: https://github.com/opensearch-project/OpenSearch/blob/cf02b9677fde73056e071c6aaae2dd67b227ffe3/buildSrc/src/main/java/org/opensearch/gradle/precommit/LicenseAnalyzer.java#L80